webportal/src/app/vc/vc.component.js

```diff
@@ -43,14 +43,15 @@ const loadData = (specifiedVc) => {
         specifiedVc: specifiedVc,
         data: data,
         formatNumber: formatNumber,
+        yarnWebPortalUri: webportalConfig.yarnWebPortalUri,
         grafanaUri: webportalConfig.grafanaUri,
       });
       $('#content-wrapper').html(vcHtml);
       table = $('#vc-table').dataTable({
         scrollY: (($(window).height() - 265)) + 'px',
         lengthMenu: [[20, 50, 100, -1], [20, 50, 100, 'All']],
         columnDefs: [
-          { type: 'natural', targets: [0, 1] },
+          { type: 'natural', targets: [0, 1, 2, 3, 4, 5, 6] },
```

Comment:
What's this for? Why before is only [0, 1]?

Reply:
It was a bug.

Reply:
For defining the method of sorting each column.